risc0/circuit/rv32im-v2-sys/kernels/cuda/defs.cu.inc

```diff
@@ -1,7 +1,7 @@
 SET_FIELD(BabyBear);
-constexpr size_t kRegCountAccum = 80;
+constexpr size_t kRegCountAccum = 103;
```

Comment:
is this additional 23 columns expected?

Reply:
Yes, this is the BigInt verification logic.